operator/pkg/utils/imageprovider/imageprovider.go

```diff
@@ -38,6 +39,7 @@ const (
 	kubeVersion122Tag = "v1.22.16-eks-1-22-14"
 	kubeVersion123Tag = "v1.23.13-eks-1-23-9"
 	kubeVersion124Tag = "v1.24.8-eks-1-24-5"
+	kubeVersion125Tag = "v1.25.5-eks-1-25-3"
```

Comment:
You might need some more logic around how/which flags are set for KCM and other components checkout -
https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/operator/pkg/controllers/master/kubecontrollermanager.go#L272